# A list of 2D detections, for a multi-object 2D detector.

Header header

Counter counter

Detector2D[] detections